homeassistant/components/switcher_kis/strings.json

```diff
@@ -63,6 +63,14 @@
       "temperature": {
         "name": "Current temperature"
       }
+    },
+    "switch": {
+      "child_lock": {
+        "name": "Lock physical controls"
```

Comment:
In the Switcher app, it is just a lock icon to lock or unlock the device meaning it will block physical controls or allow it, this is meant for child lock so the child won't accidently hit the physical controls.
I think this is the same meaning.

Reply:
I don't see how it has the same meaning. It doesn't match the docs PR

Reply:
By docs PR, do you mean this one (https://github.com/home-assistant/home-assistant.io/pull/36188)?
What is your suggestion? Should we refactor this PR to something with child lock as a translation or change the other PR?